repo/packages/M/mr-redis/0/marathon.json.mustache

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "{{mr-redis.framework-name}}",
+    "container": {
+        "docker": {
+            "image": "{{resource.assets.container.docker.redis-docker}}",
+            "network": "HOST",
+            "forcePullImage": true
+        },
+        "type": "DOCKER",
+        "volumes": []
+    },
+    "args": [],
+    "cpus": {{mr-redis.cpus}},
+    "mem": {{mr-redis.mem}},
+    "instances": 1,
```

Comment:
Shouldn't this be `mr-redis.cluster-size`? The help text suggests it is instances.